webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue

```diff
@@ -26,19 +26,85 @@
         </v-btn>
         <v-spacer />
         <exo-space-favorite-action
-          v-if="space.isMember && favoritesSpaceEnabled"
+          v-if="space.isMember && favoritesSpaceEnabled && !isMobile"
           :is-favorite="space.isFavorite"
           :space-id="space.id" />
+        <div v-else-if="isMobile">
```

Comment:
Please dissociate both conditions, it's not related and thus shouldn't use `v-else-if` here.

```suggestion
        <div v-if="!isMobile">
```